src/i18n/locales/en.json

```diff
@@ -52,6 +52,29 @@
       "DESCRIPTION": "BRL-CAD respects your freedom so our code is open source under OSI approved license terms, which means you can customize it according to your needs.It also means that you will get this software Free of cost and we won't charge you ever for any update or support."
```

Comment:
I am just asking @divyanshugarg36 where you are fetching this content from?

Reply:
Mostly from original site and wikipedia

Reply:
okay great